src/components/Content/TaskManager/TaskForm/TaskForm.css

```diff
@@ -3,6 +3,10 @@
 }
 
 .nested-form {
+  padding-left: 32px;
+}
+
+.leaf-form {
```

Comment:
@MartyTheeMartian It indents the perfect amount when the first form shows up, but then after that task is created and I press `Enter`, there's no empty box/leaf icon to the left, and thus the form is too far to the left